src/modules/wallet/components/RegisterMultisigForm/index.js

```diff
@@ -145,6 +146,30 @@ const Form = ({ nextStep, prevState = {}, onNext }) => {
     }
   }, [numberOfSignatures]);
 
+  useEffect(() => {
+    if (!isEmpty(getInitialMembersState(prevState))) {
```

Comment:
shouldnt we also return if `authQuery.isFetching`?
`if (authQuery.isFetching) {return;}`

Reply:
This already is handled below

Reply:
But why use different sources in dependency array as in effect body?